### PR TITLE
Add logout button [AK-111]

### PR DESCRIPTION
**Pull request by szarbartosz**

### CODE REVIEW

File: apps/client/pages/settings.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      <Flex w="100%" justifyContent="center">
+        <Text fontSize="4xl">{`Hello, ${data.me.name}`}</Text>
+      </Flex>
+      <VStack mt="4">
+        <CommonButton
+          variant="solid"
+          type="submit"
+          buttonText="Sign out"
+          onClick={() => {
+            logoutMutation();
```
Comment (dimamik) [inline apps/client/pages/settings.tsx:35]: Probably we need to await there?

Comment (szarbartosz) [reply apps/client/pages/settings.tsx]: good point, fixed

File: libs/moment-service/src/lib/moment-service.tsx [hunk 2]
```diff
@@ -18,8 +18,8 @@ export const deadlineToDate = (deadline: string, dateFormat = 'DD MMM') => {
   return moment(deadline, 'x').format(dateFormat);
 };
 
-export const startToDate = (start: moment.Moment, dateFormat = 'DD MMM') => {
-  return moment(start).format(dateFormat);
+export const momentToDate = (m: moment.Moment, dateFormat = 'DD MMM') => {
+  return moment(m).format(dateFormat);
```
Comment (dimamik) [inline libs/moment-service/src/lib/moment-service.tsx:22]: This needs vopy as it modifies previous value, which we probably don't need?

(Momentjs works inplace)

**Review by dimamik — APPROVED** [review]
Left some small nits, but I think we're good to go!

File: apps/client/pages/edit/user.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                label="Name"
+                touched={!!touched.name}
+              />
+            </Box>
+            <Box my="4">
+              <InputField
+                name="birthDate"
+                label="Birthdate"
+                type="date"
+                touched={!!touched.birthDate}
```
[the remark this answers is not in the document]
Comment (szarbartosz) [reply apps/client/pages/edit/user.tsx:73]: there is - it's referenced by validationSchema in 53th line

File: apps/client/pages/edit/user.tsx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  mapUserDetailsToUpdateUserMutation,
+  userToUserDetailsType,
+} from '../../services/userService';
+
+const UserDetails: React.FC = () => {
+  const router = useRouter();
+  const { data, loading } = useMeQuery();
+  const [updateUserMutation] = useUpdateUserMutation();
+
+  const onSubmit = (values: UserDetailsType) => {
```
[the remark this answers is not in the document]
Comment (szarbartosz) [reply apps/client/pages/edit/user.tsx]: good point

**Pull request merged**